doc/neps/nep-0037-array-module.rst

```diff
@@ -26,32 +26,28 @@ Why ``__array_function__`` hasn't been enough
 
 There are two broad ways in which NEP-18 has fallen short of its goals:
 
-1. **Maintainability concerns**. `__array_function__` has significant
+1. **Backwards compatibility concerns**. `__array_function__` has significant
    implications for libraries that use it:
 
-   - Projects like `PyTorch
-     <https://github.com/pytorch/pytorch/issues/22402>`_, `JAX
-     <https://github.com/google/jax/issues/1565>`_ and even `scipy.sparse
-     <https://github.com/scipy/scipy/issues/10362>`_ have been reluctant to
-     implement `__array_function__` in part because they are concerned about
-     **breaking existing code**: users expect NumPy functions like
+   - `JAX <https://github.com/google/jax/issues/1565>`_ has been reluctant
+     to implement ``__array_function__`` in part because it is concerned about
+     breaking existing code: users expect NumPy functions like
      ``np.concatenate`` to return NumPy arrays. This is a fundamental
      limitation of the ``__array_function__`` design, which we chose to allow
      overriding the existing ``numpy`` namespace.
+
+     Note that projects like `PyTorch
+     <https://github.com/pytorch/pytorch/issues/22402>`_ and `scipy.sparse
+     <https://github.com/scipy/scipy/issues/10362>`_ have also not
+     adopted ``__array_function__`` yet, because they don't have a
+     NumPy-compatible API or semantics. In the case of PyTorch, that is likely
+     to be added in the future; ``scipy.sparse`` is in the same situation as
+     ``numpy.matrix``.*
```

Comment:
Is `*` a just stray character or is intended to mark a footnote or other reference?

I think I know what you mean by referring to `numpy.matrix` (it implements different, incompatible semantics with `numpy.ndarray`) but it's not obvious from the writing here.

Reply:
thanks, fixed.